SurveyApp/Modules/SurveyQuestion/SurveyQuestionPresenter.swift

```diff
@@ -13,6 +13,9 @@ final class SurveyQuestionPresenter {
 
     weak var view: SurveyQuestionViewInput?
     weak var output: SurveyQuestionOutput?
+    
+    internal var surveyId: String?
```

Comment:
internal is the default so no need to declare here
```suggestion
    var surveyId: String?
```